_posts/2024-04-01-oidc-proxy.adoc

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: 'Use OIDC Proxy to integrate OIDC service endpoints with custom ChatGPT'
+date: 2024-04-01
+tags: extension oidc security chatgpt development-tips
+synopsis: 'Explain how OIDC Proxy can help to integrate OIDC service endpoints with custom ChatGPT'
```

Comment:
Fyi They are called GPTs not ChatGPTs.

Reply:
@maxandersen Sure, I've replaced all but a couple of references to ChatGPT, have a look please